### PR TITLE
KOGITO-5896 Configure Cypress tests to be executed as part of ui-packages maven build.

### PR DESCRIPTION
**Pull request by jstastny-cz**
https://issues.redhat.com/browse/KOGITO-5896

* Adding cypress test execution into overall maven build.
* Removing cypress-related yarn scripts from parent package.json.
* Removed `test:it` script from `runtime-tools-dev-ui-webapp` package - no tests there.
* Removed passing of npmRegistryURL property in maven plugin executions where it is unnecessary (in all except install, will be validated by the PR check).
* Removed dummy cypress tests.
* Fixed failing cypress tests.
  * In a few cases new OUIA attributes were added. 
* Fixed trusty yarn scripts not to block overall build and clean up resources
  * using concurrently script
  * making sure start-server-and-test executes scripts using yarn.

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by jstastny-cz** [comment]
@cristianonicolai please see the changes in pom.xml, if it's an acceptable solution.

File: ui-packages/pom.xml [hunk 1]
```diff
@@ -190,10 +189,20 @@
             <phase>test</phase>
             <configuration>
               <skip>${skip.ui.build}</skip>
-              <npmRegistryURL>${env.NPM_REGISTRY_URL}</npmRegistryURL>
               <arguments>run test</arguments>
             </configuration>
           </execution>
+          <execution>
+            <id>yarn integration test</id>
```
Comment (cristianonicolai) [inline ui-packages/pom.xml]:
```suggestion
            <id>yarn-integration-test</id>
```

**Comment by jstastny-cz** [comment]
Reaching system file limit in GH Actions :/ 
`error from chokidar limit for number of file watchers reached`

In jenkins there's an extra `--registry` parameter passed through lerna to called script, which does not know how to handle it. Resulting in failure:
`lerna run test:it --stream --concurrency=1 --no-bail -- --registry=<replaced-registry-url>`
`lerna info Executing command in 3 packages: "yarn run test:it --registry=<replaced-registry-url>"`
```
[2021-09-14T15:07:19.544Z] [INFO] @kogito-apps/management-console-webapp: $ start-server-and-test 'yarn dev' 9000 'yarn cypress:run' --registry=<replaced-registry-url>
[2021-09-14T15:07:20.108Z] [INFO] @kogito-apps/management-console-webapp: /home/jenkins/workspace/KIE/kogito/pullrequest/main/kogito-apps.tests.apps/kogito-apps/ui-packages/node_modules/lazy-ass/index.js:117
[2021-09-14T15:07:20.108Z] [INFO] @kogito-apps/management-console-webapp:       throw err;
[2021-09-14T15:07:20.108Z] [INFO] @kogito-apps/management-console-webapp:       ^
[2021-09-14T15:07:20.108Z] [INFO] @kogito-apps/management-console-webapp: Error: expected <NPM script name that starts server> <url or port> <NPM script name that runs tests>
```

File: ui-packages/pom.xml (hunk 1 above)
Comment (jstastny-cz) [reply ui-packages/pom.xml]: I did update according the proposal.

**Comment by jstastny-cz** [comment]
Converted to Draft to first resolve the builds issues.

**Comment by jstastny-cz** [comment]
Fail in jenkins - the settings.xml used contains property `npmRegistryURL` that gets promoted into frontend-maven-plugin all the time causing issues in the test:it script.

**Comment by jstastny-cz** [comment]
jenkins retest this

**Comment by jstastny-cz** [comment]
jenkins retest this

**Comment by jstastny-cz** [comment]
jenkins retest this

**Comment by jstastny-cz** [comment]
jenkins retest this

**Review by sjamboro — APPROVED** [review]
Thank you

**Review by prashuts — APPROVED** [review]

**Comment by jstastny-cz** [comment]
@AjayJagan @Sara4994 @pefernan if I may ask one of you to review the changes. I think it's safe to focus just on changes in ProcessListTable and ProcessListChildTable components, where I did some minor refactoring to introduce OUIA attributes.
If you check snapshots for those components, just changes in OUIA attributes handling should be there.

**Comment by pefernan** [comment]
@jstastny-cz sure we'll take a look, thank you and sorry for the delay

**Review by AjayJagan — APPROVED** [review]
thanks @jstastny-cz  :+1:

**Review by pefernan — APPROVED** [review]
+1 thanks @jstastny-cz

**Review by Sara4994 — APPROVED** [review]

**Comment by Sara4994** [comment]
@jstastny-cz LGTM. thanks

**Pull request merged**